lib/unimatrix/authorization/filters/requires_policies.rb

```diff
@@ -80,7 +80,8 @@ def policies
     end
 
     def retrieve_policies( resource_name, access_token, realm )
-      if resource_name && access_token && realm
+      if resource_name && access_token
+        realm    = realm || '*'
```

Comment:
Ok I think this is fine for querying, however for creating a new realm it wouldn't be checking the right permission right? Because the way it works now, if you try to write a realm, the policies before filter will find all your permissions for every realm and if you have the write action for any of them, it will work. But if you only have access to one realm you shouldn't be able to create new ones or update other ones.